docs/CSS.md

```diff
@@ -0,0 +1,125 @@
+### 1. `max-width`
+
+* elements: all
+* inherited: no
+
+#### Values
+
+* `none` No max width, this is the default
+* `length` defines max width in px, cm, etc..
+* `%` defines max width in percent of containing block
```

Comment:
One of the things that tripped up the last class (and I'm sure will recur here) is what the percentage values were a percentage _of_... Good thing to keep in mind as you work with percentages.